app/src/main/java/ch/epfl/sdp/musiconnect/Page.java

```diff
@@ -114,4 +152,14 @@ protected void signOut() {
                     finish();
                 });
     }
+
+    protected void sendNotificationToMusician(String channel, int priority) {
+        String notificationMessage = "A musician is within " + DISTANCE + " meters";
+        if (DISTANCE <= 100) {

```

Comment:
where do you change the distance value ?

If 100 is a static value, i think it will be preferrable to store it in a static final variable

Reply:
It is old code, I removed it and created a static DISTANCE_LIMIT variable equal to 200 for now.